src/CQRS/LeanCode.CQRS.MassTransitRelay/ConsumerConfigurationExtensions.cs

```diff
@@ -0,0 +1,37 @@
+using LeanCode.CQRS.MassTransitRelay.Middleware;
+using MassTransit;
+
+namespace Leancode.CQRS.MassTransitRelay;
+
+public static class ConsumerConfigurationExtensions
+{
+    public static ISendPipeConfigurator ConfigureSendActorIdPropagation(ISendPipeConfigurator cfg)
+    {
+        cfg.UseExecute(ex =>
+        {
+            if (
+                ex.TryGetPayload<ConsumeContext>(out var payload)
+                && payload.TryGetHeader<string>(EventsPublisherMiddleware.EventActorIdentifier, out var actorId)
```

Comment:
Can we, like, move it out of the `EventsPublisherMiddleware`? This does not look like a correct place for that, as propagating this is rather separate feature.